lib/mongoid/persistable/upsertable.rb

```diff
@@ -12,13 +12,13 @@ module Upsertable
       #
       # If the replace option is true, unspecified attributes will be dropped,
       # and if it is false, unspecified attributes will be maintained. The
-      # replace option defaults to false in Mongoid 9.
+      # replace option defaults to true in Mongoid 8.1 and earlier.
```

Comment:
Does it worth mentioning that the default will be changed to `false` in 9.0, since we know that?

Reply:
good call